page_creation.rst

```diff
@@ -174,58 +279,17 @@ So what about the other directories in the project?
     The famous ``bin/console`` file lives here (and other, less important
     executable files).
 
-``tests/``
-    The automated tests (e.g. Unit tests) for your application live here.
-
 ``var/``
     This is where automatically-created files are stored, like cache files
-    (``var/cache/``), logs (``var/logs/``) and sessions (``var/sessions/``).
+    (``var/cache/``) and logs (``var/logs/``).
```

Comment:
Be careful!! `var/logs/` is now called `var/log/` See https://github.com/symfony/recipes/pull/169

Reply:
Oh! I hadn't even realized! Thanks :)

Reply:
do you need some hand? see https://github.com/symfony/symfony-docs/pull/8547 ;)